tests/helpers.js

```diff
@@ -0,0 +1,15 @@
+import supertest from 'supertest';
```

Comment:
Só sendo chato com o nome [helpers.js](https://softwareengineering.stackexchange.com/questions/247267/what-is-a-helper-is-it-a-design-pattern-is-it-an-algorithm). Quase sempre é um code smell.